### PR TITLE
[cci] centos:8 support

### PR DESCRIPTION
**Pull request by gquintard**
@slimhazard , here you go :-)

### CODE REVIEW

**Comment by nigoroll** [comment]
isn't this something which can be merged straight away?

File: .circleci/config.yml [hunk 1]
```diff
@@ -146,7 +92,70 @@ jobs:
...
+                yum install -y \
+                    automake \
+                    jemalloc-devel \
+                    libconfig-devel \
+                    libcurl-devel \
+                    libedit-devel \
+                    libtool \
+                    libunwind-devel \
+                    make \
+                    openssh-clients \
```
Comment (nigoroll) [inline .circleci/config.yml]: I think we do not need
* libconfig
* libcurl
* openssh-client

**Comment by slimhazard** [comment]
I'm assuming that in the centos8 build, installing epel-release will result in the EPEL repo pointing to el8 packages, in particular jemalloc 5.2.1. And in the end ldd varnishd will show libjemalloc.so.2 (rather than libjemalloc.so.1 as currently for el7).

@gquintard do you happen to know if that's correct?

Asking because currently I'm working on getting jemalloc 5.2.1 to install on centos7, and am hoping for a beautiful centos8 future in which that all Just Works.

File: .circleci/config.yml (hunk 1 above)
Comment (gquintard) [reply .circleci/config.yml]: indeed

**Comment by gquintard** [comment]
@slimhazard, correct:

```
# rpm -qpR rpms/x86_64/varnish-20200203.weekly-0.0.el8.x86_64.rpm | grep jemalloc
libjemalloc.so.2()(64bit)
```

```
# ldd /usr/sbin/varnishd | grep jemalloc
	libjemalloc.so.2 => /lib64/libjemalloc.so.2 (0x00007f5fb7ff1000)
```

**Comment by gquintard** [comment]
looks like this got in before I could clean the PR, closing

**Comment by nigoroll** [comment]
@gquintard I do not think anyone has merged your PR.
I just updated the docs in f5dcaeabba2525c5609b66001f4743ba2fbbf7f8 / 0e192d6fa1691ebe9a4a3dcb600579cd43a0c1ba while I was working on a CentOS 8 installation anyway

**Comment by gquintard** [comment]
https://github.com/varnishcache/varnish-cache/commit/30453b94901d3ccaa2230dc0a9a477fbcb7c92af appears to be in master though

**Comment by nigoroll** [comment]
oh right. Wasn't that you?

**Comment by gquintard** [comment]
I don't think so, but that's a possibility

**Pull request closed without merging**